google/cloud/pubsublite/internal/partition_publisher.h

```diff
@@ -46,7 +48,7 @@ class PartitionPublisher
           StreamInitializer<google::cloud::pubsublite::v1::PublishRequest,
                             google::cloud::pubsublite::v1::PublishResponse>)>,
       BatchingOptions, google::cloud::pubsublite::v1::InitialPublishRequest,
-      AlarmRegistry&);
+      PublishSequenceNumber, AlarmRegistry&);
```

Comment:
it seems that only tests initialize `PulishSequenceNumber` with something other than 0.  Consider making this the last argument and maybe passing 0 by default.